uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1002.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1002 {
+
+  /**
+   * Creates the 'other' sickness reason option value.
+   *
+   * @return bool
+   */
+  public function upgrade_1002() {
+    try {
+      $result = civicrm_api3('OptionValue', 'get', [
+        'sequential' => 1,
+        'option_group_id' => 'hrleaveandabsences_sickness_reason',
+        'name' => 'other',
+      ]);
+
+      if ($result['count'] == 0) {
+        civicrm_api3('OptionValue', 'create', [
+          'option_group_id' => 'hrleaveandabsences_sickness_reason',
+          'name' => 'other',
+          'label' => 'Other - Please leave a comment',
+          'value' => 12,
+          'weight' => 11,
+          'is_reserved' => 1,
+          'is_default' => 0,
+          'is_active' => 1
+        ]);
+      }
+    } catch(Exception $e) {
+      // We run all the upgraders during the extension installation, but, during
```

Comment:
The one added to the xml only gets to run when L&A extension is installed. There are sites already where L&A is installed, so its not possible to uninstall and re-install to add the option value so thats the reason for adding it to the upgrader also. 
For a fresh install, since we run all upgraders during install, for some reason, Please check this [PR](https://github.com/civicrm/civihr/pull/1996), the option group is not available and an error is thrown, but the option values will install just fine because it will install via the xml file.